mxnet/inference/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -40,7 +42,11 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
     LANG=C.UTF-8 \
     LC_ALL=C.UTF-8 \
     TEMP=/home/model-server/tmp \
-    DEBIAN_FRONTEND=noninteractive
+    DEBIAN_FRONTEND=noninteractive \
+    DLC_CONTAINER_TYPE=inference
```

Comment:
What is the reason for adding this `env` variable?

Reply:
This resolves the issue that mxnet inference images also report as training images. This enables the MXNet binary to know what type of image it is operating in.